lambdas/issuecredential/src/main/java/uk/gov/di/ipv/cri/drivingpermit/api/handler/IssueCredentialHandler.java

```diff
@@ -179,20 +181,15 @@ public APIGatewayProxyResponseEvent handleRequest(
                     HttpStatusCode.BAD_REQUEST, ErrorResponse.VERIFIABLE_CREDENTIAL_ERROR);
         } catch (SqsException sqsException) {
             LOGGER.error(
-                    "Exception while handling lambda {} exception {}",
-                    context.getFunctionName(),
-                    sqsException.getClass());
+                    LAMBDA_HANDLING_EXCEPTION, context.getFunctionName(), sqsException.getClass());
             eventProbe.counterMetric(LAMBDA_ISSUE_CREDENTIAL_COMPLETED_ERROR);
 
             LOGGER.debug(sqsException.getMessage(), sqsException);
 
             return ApiGatewayResponseGenerator.proxyJsonResponse(
                     HttpStatusCode.INTERNAL_SERVER_ERROR, sqsException.getMessage());
         } catch (Exception e) {
-            LOGGER.error(
```

Comment:
It best not log whole exceptions unless we known the exception type and its contents.
The issue with this in production is that if there is an null pointer or an exception not caught where it should be, it will end up here and wrote out to the logs. If the code happened to be processing something with PII in it that will appear also.

The other side of this is in dev we want these stack traces , there is a debug level out on line 195.

The line should suppress the exception at none debug levels
`LOGGER.error("Exception while handling lambda {}", context.getFunctionName());`